screens/CheckoutScreen.js

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+import { BASE } from "../lib/common";
+
+import {
+  AsyncStorage,
+  StyleSheet,
+  TextInput,
+  View,
+  Button,
+  Text
+} from "react-native";
+
+export default class CheckoutScreen extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      customerName: "",
+      customerId: 0,
+      currPoints: 0,
+      cart: [],
+      totalPrice: 0,
+      totalPoints: 0
+    };
+  }
+
+  async componentDidMount() {
+    const customerId = await AsyncStorage.getItem("customerId");
+    this.setState({ customerId: customerId });
+
+    this.getUser("Customers", customerId).then(customerRecord => {
+      if (customerRecord) {
+        let currPoints = customerRecord["fields"]["points"];
+        let name =
+          customerRecord["fields"]["First Name"] +
+          " " +
+          customerRecord["fields"]["Last Name"];
+        this.setState({ customerName: name, currPoints: currPoints });
+      }
+    });
+  }
+
+  async getUser(table, id) {
```

Comment:
so for refactoring, anything that doesn't call `this.setState` can and should generally be extracted into helper files like `authAirtable.js` or `storeHelpers.js` in the other repo (lol I have 0 consistency in naming)

Reply:
Removing this file

Reply:
Will probably move methods to helper files in a different PR.